src/Libraries/Microsoft.Extensions.Hosting.Testing/FakeHost.cs

```diff
@@ -2,16 +2,19 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Diagnostics.CodeAnalysis;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Hosting;
+using Microsoft.Shared.DiagnosticIds;
 using Microsoft.Shared.Diagnostics;
 
 namespace Microsoft.Extensions.Hosting.Testing;
 
 /// <summary>
 /// Unit testing friendly configured host.
 /// </summary>
+[Experimental(diagnosticId: Experiments.Hosting, UrlFormat = Experiments.UrlFormat)]
```

Comment:
@geeknoid can you confirm that this is correct?

Reply:
Note these were added to match the manifest. If it's wrong, we'll need to update the manifest.
https://github.com/dotnet/extensions/blob/6fda2c8b8f8bacf74eaafff243a10ea20f849aaa/src/Libraries/Microsoft.Extensions.Hosting.Testing/Microsoft.Extensions.Hosting.Testing.json#L5-L6

Reply:
Got it, I just want to make sure we understand which one is correct, manifest or code. Given this is a Fake, it wasn't immediately obvious to me why this wouldn't be stable.

Reply:
Oh actually, this is all working at expected.

The whole package is experimental as we have Stage=dev in the .csproj. Thus, the experimental attribute is being applied at the level of the assembly and doesn't need to be applied to each type/member. So you can safely remove this experimental annotation.

Reply:
I see, that makes sense. I didn't know that we had infrastructure setup for injecting the attribute to the assembly but it does seem to be working then.